packages/utilities/psammead-storybook-helpers/src/index.test.jsx

```diff
@@ -36,71 +42,178 @@ describe('Psammead storybook helpers', () => {
-      beforeAll(() => {
-        mockBooleanResult = true;
-      });
+    it('displays english snippet when no slot default text provided', () => {
+      const english = {
+        text: 'Could a computer ever create better art than a human?',
+        script: 'latin',
+        dir: 'ltr',
+      };
+      select.mockReturnValueOnce(english);
+      text.mockImplementation((_, displayText) => displayText);
 
-      it('calls the render function with one input', () => {
-        underTest.inputProvider(['first'], renderFn)();
+      underTest.inputProvider([{ name: 'first' }], renderFn)();
 
-        expect(renderFn).toHaveBeenCalledTimes(1);
-        expect(renderFn).toHaveBeenCalledWith('select knob');
+      expect(renderFn).toHaveBeenCalledTimes(1);
+      expect(renderFn).toHaveBeenCalledWith(
+        ['Could a computer ever create better art than a human?'],
+        'LATIN SCRIPT OBJECT',
+        'ltr',
+      );
+
+      expect(select).toHaveBeenCalledTimes(1);
+      expect(text).toHaveBeenCalledTimes(1);
+    });
 
-        expect(boolean).toHaveBeenCalledTimes(1);
-        expect(select).toHaveBeenCalledTimes(1);
-        expect(text).toHaveBeenCalledTimes(0);
-      });
+    it('defaults to language text for non-english languages', () => {
+      const russian = {
+        text:
+          'Мнение Назарбаева будет иметь приоритетное значение при принятии важных для страны решений, сказал Токаев',
+        script: 'cyrillic',
+        dir: 'ltr',
+      };
+      select.mockReturnValueOnce(russian);
+      text.mockImplementation((_, displayText) => displayText);
+
+      underTest.inputProvider(
+        [{ name: 'first', defaultText: 'Sole input' }],
+        renderFn,
+      )();
+
+      expect(renderFn).toHaveBeenCalledTimes(1);
+      expect(renderFn).toHaveBeenCalledWith(
+        [
+          'Мнение Назарбаева будет иметь приоритетное значение при принятии важных для страны решений, сказал Токаев',
+        ],
+        'CYRILLIC SCRIPT OBJECT',
+        'ltr',
+      );
+    });
+
+    describe('passes text direction', () => {
+      it('defaulting to ltr when not specified', () => {
+        const russian = {
+          script: 'cyrillic',
+        };
+        select.mockReturnValueOnce(russian);
+        text.mockImplementation((_, displayText) => displayText);
 
-      it('calls the render function with multiple inputs', () => {
-        underTest.inputProvider(['first', 'second'], renderFn)();
+        underTest.inputProvider([], renderFn)();
 
         expect(renderFn).toHaveBeenCalledTimes(1);
-        expect(renderFn).toHaveBeenCalledWith('select knob', 'select knob');
+        expect(renderFn).toHaveBeenCalledWith(
+          [],
+          'CYRILLIC SCRIPT OBJECT',
+          'ltr',
+        );
+      });
 
-        expect(boolean).toHaveBeenCalledTimes(2);
-        expect(select).toHaveBeenCalledTimes(2);
-        expect(text).toHaveBeenCalledTimes(0);
+      it('when specified', () => {
```

Comment:
These `it` names need a tad reword. Maybe

Text Direction
- defaults to ltr when not specified
- returns correct value when specified